frontend/packages/dev-console/src/components/pipelines/PipelinesPage.tsx

```diff
@@ -22,7 +29,7 @@ const PipelinesPage: React.FC<PipelinesPageProps> = ({ namespace }) => {
       filters: { ...filters },
     },
   ];
-  return namespace ? (
+  return namespace || perspective !== 'dev' ? (
```

Comment:
@christianvogt @andrewballantyne I'm not really happy with how I've addressed this... Let me know if you can think of a better way.

Reply:
I think this is a dev-console mistake. I logged https://jira.coreos.com/browse/ODC-2095 to address our overriding of the resource without gating it behind a perspective.

Our `dev-console/src/plugin.tsx` showcases overriding build configs:
```
  {
    type: 'Page/Route',
    properties: {
      perspective: 'dev',
      exact: true,
      path: ['/k8s/all-namespaces/buildconfigs', '/k8s/ns/:ns/buildconfigs'],
      loader: async () =>
        (await import(
          './components/BuildConfigPage' /* webpackChunkName: "dev-console-buildconfigs" */
        )).default,
    },
  },
```
I feel we should do something like that here.

@christianvogt, thoughts?

Reply:
This still work fix the search page in the dev perspective ;/
We may need to create standard list pages and then add special links to the dev console perspective to hit the customized list pages.
I'd need to think about it more.

Reply:
I think this is going to be technical debt that we address in ODC-2095 - I am not sure it's worth holding up / considering anything else for this PR. Unless @christianvogt has a master plan in his back pocket, that is.

Reply:
We can address in a bug later. It already existed if we used the search in dev perspective.